bitsandbytes/functional.py

```diff
@@ -421,41 +417,36 @@ def create_quantile_map(A, total_bits=8):
     return q
 
 
+@deprecated("This function is deprecated and will be removed in a future version.", category=FutureWarning)
 def get_special_format_str():
-    if not torch.cuda.is_available():
-        return "col_turing"
-    major, _minor = torch.cuda.get_device_capability()
-    if major <= 7:
-        return "col_turing"
-    if major == 8:
-        return "col_ampere"
-    return "col_turing"
+    return "row"
 
 
-def is_on_gpu(tensors):
+def is_on_gpu(tensors: Iterable[torch.Tensor]):
```

Comment:
Since you check for `t is not None`, I think the signature should note that nones are OK:
```suggestion
def is_on_gpu(tensors: Iterable[Optional[torch.Tensor]]):
```